src/Microsoft.Restier.WebApi/Query/QueryTotalCount.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Restier.Core.Query;
+
+namespace Microsoft.Restier.WebApi.Query
+{
+    internal class QueryTotalCount : IQueryExecutor
+    {
```

Comment:
@lewischeng-ms I was thinking we may have publisher like OneAPi,OpenAPI and so on, then we  may have OneAPIQueryOptions, OpenAPIQueryOptions and so on, and this is the reason I suggest name OData as prefix, if we want to differentiate from low level ODL lib class names, I would suggest RestierODataQueryExecutor, then in future we may have RestierOneAPIQueryExecutor,RestierOpenApiQueryExecutor, comments?